features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml.j2

```diff
@@ -300,70 +354,81 @@
         <Scopes>
-            <Scope displayName="View Notification Senders" name="internal_notification_senders_view"/>
-            <Scope displayName="Create Notification Senders"
-                   name="internal_notification_senders_create"/>
-            <Scope displayName="Update Notification Senders"
-                   name="internal_notification_senders_update"/>
-            <Scope displayName="Delete Notification Senders"
-                   name="internal_notification_senders_delete"/>
+            <Scope displayName="View Notification Senders" name="internal_notification_senders_view" 
+                    description="View new notification senders for the organization (root)"/>
+            <Scope displayName="Create Notification Senders" name="internal_notification_senders_create" 
+                    description="Create notification senders for the organization (root)"/>
+            <Scope displayName="Update Notification Senders" name="internal_notification_senders_update" 
+                    description="Update notification senders for the organization (root)"/>
+            <Scope displayName="Delete Notification Senders" name="internal_notification_senders_delete" 
+                    description="Delete notification senders for the organization (root)"/>
         </Scopes>
     </APIResource>
     <APIResource name="Notification Sender Management API"
                  identifier="/o/api/server/v1/notification-senders" requiresAuthorization="true"
                  description="API representation of the Notification Sender Management API" type="ORGANIZATION_ADMIN">
         <Scopes>
-            <Scope displayName="View Notification Senders" name="internal_org_notification_senders_view"/>
-            <Scope displayName="Create Notification Senders"
-                   name="internal_org_notification_senders_create"/>
-            <Scope displayName="Update Notification Senders"
-                   name="internal_org_notification_senders_update"/>
-            <Scope displayName="Delete Notification Senders"
-                   name="internal_org_notification_senders_delete"/>
+            <Scope displayName="View Notification Senders" name="internal_org_notification_senders_view" 
+                    description="View new notification senders for organization"/>
+            <Scope displayName="Create Notification Senders" name="internal_org_notification_senders_create" 
+                    description="Create new notification senders for organization"/>
+            <Scope displayName="Update Notification Senders" name="internal_org_notification_senders_update" 
+                    description="Update new notification senders for organization"/>
+            <Scope displayName="Delete Notification Senders" name="internal_org_notification_senders_delete" 
+                    description="Delete new notification senders for organization"/>
```

Comment:
`new` should be removed for view, delete and update

Reply:
Removed